app/models/pipeline_run.rb

```diff
@@ -1109,13 +1109,38 @@ def update_job_status
         # Check for long-running pipeline run and log/alert if needed
         check_and_log_long_run
```

Comment:
What happened to checks for long runs?

Reply:
Removed on intent. It doesn't make sense to check for long runs here, since this method will be invoked asynchronously, therefore it means we had some progress. Checking for long runs need to be done through a polling mechanism, timeouts in the step function, or both. It is more related to ticket IDSEQ-2378.

Reply:
We could trigger timeout events and keep the logic asynchronous instead of having a separate poll mechanism.
However, since it seems we can poll from the DB just runs that have not been updated by using updated_at, i am not too worried.